estate/models/estate_property.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "property"
+    _description = """
+    This model provides a representation of a real estate property, including important fields such as the property name, description, postcode, date availability, expected price, selling price, number of bedrooms, living area, number of facades, and other features such as garage and garden.
```

Comment:
This description is Okay but its just too long, we can shorten it like this:

```suggestion
Representation of a real estate property
```

The fields itsel are self explanatory so, there is no need to actually telling here all the fields